INSTALL.md

```diff
@@ -0,0 +1,269 @@
+#Installation Instructions
+
+OSI Certified Open Source Software
+
+-------------------------------------
+
+### Table of Contents
+
+**[Overview of Directories](#overview-of-directories)**
+
+**[Unpacking](#unpacking)**
+
+**[Setup](#setup)**
+
+**[Setting up Access Control](#setting-up-access-control)**
+
+**[Upgrading](#upgrading)**
+
+**[FAQ](#faq)**
+
+
+##  Overview of Directories
+
+NOTE: Most recent documentation can be found on the online documentation at [OpenEMR](http://www.open-emr.org).
+
+accounting: Contains information and scripts to support SQL-Ledger
+contrib: Contains many user-contributed encounter forms and utilities
+custom: Contains scripts and other text files commonly customized
+Documentation: Contains useful documentation
+interface: Contains User Interface scripts and configuration
+library: Contains scripts commonly included in other scripts
+sql: Contains initial database images
+gacl: Contains embedded php-GACL (access controls)
+
+##  Unpacking
+
+NOTE: Most recent documentation can be found on the online documention at [OpenEMR](http://www.open-emr.org).
+
+The LibreEHR release archive should be named as follows:
+
+`libreehr-<version>.tar.gz  -or-  libreehr-<version>.zip`
```

Comment:
I think the packages will need to be named LibreHealthEHR.  LibreEHR is a nickname, but the LibreHealth community at large would like us to use LibreHealthEHR as much as possible.